libs/datamodel/connectors/sql-datamodel-connector/src/mssql_datamodel_connector.rs

```diff
@@ -249,12 +249,6 @@ impl Connector for MsSqlDatamodelConnector {
                     Float(Some(bits)) if bits == 0 || bits > 53 => {
                         error.new_argument_m_out_of_range_error("Bits can range from 1 to 53.")
                     }
-                    typ if heap_allocated_types().contains(&typ) && field.is_unique() => {
-                        error.new_incompatible_native_type_with_unique()
```

Comment:
where did this go?

Reply:
These are now model level. They are now only checked there. Before they were checked both on model and field level